applications/DEMApplication/python_scripts/sphere_strategy.py

```diff
@@ -689,11 +689,12 @@ def ModifyProperties(self, properties, param = 0):
         if properties.Has(FRICTION):
             self.Procedures.KratosPrintWarning("-------------------------------------------------")
             self.Procedures.KratosPrintWarning("  WARNING: Property FRICTION is deprecated since April 6th, 2020, ")
-            self.Procedures.KratosPrintWarning("  replace with STATIC_FRICTION and DYNAMIC_FRICTION")
+            self.Procedures.KratosPrintWarning("  replace with STATIC_FRICTION, DYNAMIC_FRICTION and FRICTION_DECAY")
             self.Procedures.KratosPrintWarning("  Automatic replacement is done now.")
             self.Procedures.KratosPrintWarning("-------------------------------------------------")
             properties[STATIC_FRICTION] = properties[FRICTION]
             properties[DYNAMIC_FRICTION] = properties[FRICTION]
+            properties[FRICTION_DECAY]  = 0.0
```

Comment:
Shouldn't this be a 500? 500 is an acceptable value in case the user does not put any value. I would put the default value here (500, 200 or whatever it is). Also, in those cases where there is no difference between dynamic and static friction, I would not write FRICTION_DECAY, or if I write it, I would give it the default value as well. 
If we think of a user who just changes a couple of values here ant there to create a new case, it is probable that he changes the DYNAMIC_FRICTION but forgets about FRICTION_DECAY. In that case, an absent FRICTION_DECAY is better, or a FRICTION_DECAY with a realistic value (500).

Reply:
You are right, I will change that. What do you think about the tests? Should I add FRICTION_DECAY everywhere? Maybe, it could be a good idea to add a new test to check this feature out and leave the previous tests without FRICTION_DECAY, what do you think?

Reply:
Yes, we can leave the old tests without DECAY. It's also a way to make sure that we set the default value correctly.